posts/inside-rust/2023-01-24-content-delivery-networks.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Diversifying our Content Delivery Networks
+author: Jan David Nose
+team: The Rust Infrastructure Team <https://www.rust-lang.org/governance/teams/infra>
+---
+
+Over the past few weeks, the [Infrastructure Team] has been working on a setting
+up a second [Content Delivery Network] for releases and crates.
+
+## What is changing
+
+Rust releases (`static.rust-lang.org`) and crates (`static.crates.io`) will be
+served through both [AWS CloudFront](https://aws.amazon.com/cloudfront/) and
+[Fastly](https://www.fastly.com).
+
+This is a transparent change to the infrastructure that doesn't require users or
+developers to take any action. We expect no downtime for Rust users as part of
+the migration.
+
+This also covers only the CDNs for releases (`static.rust-lang.org`) and
+crates.io (`static.crates.io`), no changes are planned for the rest of Rust's
+domains.
+
+## Timeline
+
+We are planning to start the rollout of the new CDN on Tuesday, January 24rd,
```

Comment:
We might want to just say "today, January 24th" so that folks don't need to realize that it's effectively already happening. 
```suggestion
We are planning to start the rollout of the new CDN on Tuesday, January 24th,
```